logunit-jul/src/test/java/io/github/netmikey/logunit/jul/LogCapturerWithJulTest.java

```diff
@@ -83,4 +85,10 @@ private void logEverythingOnce(Logger logger) {
         logger.warning("Some warning message");
         logger.severe("Some severe message");
     }
+
+    // The logger will not be until the object is. This tests that log messages will be captured without allowing
```

Comment:
There seems to be a word missing here(?)

Reply:
Oops fixed the comment